addons/calendar/models/calendar.py

```diff
@@ -318,6 +318,10 @@ def do_check_alarm_for_one_date(self, one_date, event, event_maxdelta, in_the_ne
         return result
 
     @api.model
+    def send_event_reminders(self):
```

Comment:
It's to make it generic, in case other style of reminders get added (like sms reminders ;) )